infinidoko/week1/fizzbuzz.txt

```diff
@@ -0,0 +1,22 @@
+//fizzbuzz program by David Idajili
+#include<iostream>
+using namespace std;
+int main(){
+       //loop to iterate from 1 to 100
+	for(int i = 1; i<=100; i++){
+		
+		if((i%3 == 0)&&(i%5 == 0))           //check if i is both a multiple of 3 and 5
+			cout<<"FizzBuzz"<<"\n";     //print FizzBuzz 
+

```

Comment:
Kindly delete this file and update you Pull request. It is a duplicate of what is in `fizzbuzz.cpp`